fs-diff-test-utils/Dockerfile.podman-image

```diff
@@ -0,0 +1,3 @@
+FROM centos:8.3.2011
```

Comment:
Any reason you aren't using `ubi` instead?

Reply:
Couldn't install podman on ubi.